<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moRFeus_Qt</class>
 <widget class="QMainWindow" name="moRFeus_Qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>369</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>369</width>
    <height>287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>moRFeus_Qt</string>
  </property>
  <widget class="QWidget" name="moRFeus">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>369</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>369</width>
     <height>250</height>
    </size>
   </property>
   <widget class="QLabel" name="startlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Frequency</string>
    </property>
    <property name="buddy">
     <cstring>startfreq</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End Frequency</string>
    </property>
    <property name="buddy">
     <cstring>endfreq</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>72</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step KHz</string>
    </property>
    <property name="buddy">
     <cstring>stepInput</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>102</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
    <property name="buddy">
     <cstring>powerInput</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="stepInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.001000000000000</double>
    </property>
    <property name="maximum">
     <double>50000.000000000000000</double>
    </property>
    <property name="value">
     <double>250.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="endfreq">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>85.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5400.000000000000000</double>
    </property>
    <property name="value">
     <double>88.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="startfreq">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>85.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5400.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="powerInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="time">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sweep Time</string>
    </property>
    <property name="buddy">
     <cstring>time</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>162</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Signal delay ms</string>
    </property>
    <property name="buddy">
     <cstring>delay</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="delay">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="noisebutton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Noise</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>193</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Morse Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morsebutton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Msg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="biason">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bias On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genbutton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="biasoff">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bias Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sweepbutton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sweep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mixbutton">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mixer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="morseInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuMoRFeus">
    <property name="title">
     <string>MoRFeus</string>
    </property>
    <addaction name="actionVersion_0_370"/>
   </widget>
   <addaction name="menuMoRFeus"/>
  </widget>
  <action name="actionVersion_0_369">
   <property name="text">
    <string>Version 0.369</string>
   </property>
  </action>
  <action name="actionOpen_SDRplot">
   <property name="text">
    <string>Open SDRplot</string>
   </property>
  </action>
  <action name="actionVersion_0_370">
   <property name="text">
    <string>Version 0.669</string>
   </property>
  </action>
  <action name="actionMessenger">
   <property name="text">
    <string>Messenger</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>startfreq</tabstop>
  <tabstop>endfreq</tabstop>
  <tabstop>stepInput</tabstop>
  <tabstop>powerInput</tabstop>
  <tabstop>time</tabstop>
  <tabstop>delay</tabstop>
  <tabstop>mixbutton</tabstop>
  <tabstop>genbutton</tabstop>
  <tabstop>noisebutton</tabstop>
  <tabstop>sweepbutton</tabstop>
  <tabstop>biason</tabstop>
  <tabstop>biasoff</tabstop>
  <tabstop>morsebutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
